ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-SVC3A.cfg

```diff
@@ -874,9 +874,9 @@ ip prefix-list PL-MLAG-PEER-VRFS
    seq 10 permit 10.255.251.6/31
 !
 monitor session MonitoringSessionServer18WithDest mac access-group mac_acl
-monitor session MonitoringSessionServer18WithDest source Ethernet25 rx ip access-group MyIpACL priority 5
-monitor session MonitoringSessionServer18WithDest source Ethernet28 tx mac access-group MyMacACL priority 5
-monitor session MonitoringSessionServer18WithDest source Port-Channel27 tx mac access-group MyMacACL priority 5
+monitor session MonitoringSessionServer18WithDest source Ethernet25 rx
```

Comment:
Please add another session so we still have tests of mac access groups and both ip and mac with priority.

Reply:
Also update the test data with comments saying that these options are ignored because of the ACL on the session.

Reply:
Added.